live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/adapters/config/properties/Repository.java

```diff
@@ -27,5 +27,5 @@
 
 import jakarta.validation.constraints.NotBlank;
 
-public record Repository(@NotBlank String endpoint, @NotBlank String discoveryEndpoint)
+public record Repository(@NotBlank String endpoint, @NotBlank String discoveryEndpoint, String version)
```

Comment:
We could call this `versionOverride` to make it clear that it's an alternative to using `discoveryEndpoint`. Should `discoveryEndpoint` still be annotated `@NotBlank`?

Reply:
Done. For now I removed the `@NotBlank` annotation for `discoveryEndpoint`. I think we can implement validation described by @dsibilio in this thread: https://github.com/Alfresco/hxinsight-connector/pull/516#discussion_r1691144260